qbft/spectest/tests/proposal/current_round_prev_prepared.go

```diff
@@ -35,7 +35,7 @@ func CurrentRoundPrevPrepared() *tests.MsgProcessingSpecTest {
 	return &tests.MsgProcessingSpecTest{
 		Name:          "proposal happy flow round > 1 (prev prepared)",
 		Pre:           pre,
```

Comment:
the round change justifications have dataRound == NoRound. 
This basically means that it's not a previously prepared proposal with previously prepared round changes.
If you do step by step you'll see it goes into proposal.go#L153

Reply:
https://github.com/bloxapp/ssv-spec/pull/157#discussion_r1114120183